TESTS/psa/crypto_init/main.cpp

```diff
@@ -102,5 +98,13 @@ Specification specification(greentea_test_setup, cases, greentea_test_teardown_h
 
 int main()
 {
+    uint8_t seed[MBEDTLS_PSA_INJECT_ENTROPY_MIN_SIZE] = {0};
+    /* inject some a seed for test*/
```

Comment:
This is only needed if the seed hasn't been injected at the factory yet. Factory tool should do this, not our tests, right?

Reply:
but then the tests won't work. since there is no prior factory injection for tests

Reply:
tests are expected to work during OOB - so we must inject entropy in tests.
as a side effects tests executed on a device may destroy/corrupt factory values - but this is true for all the tests. KVStore tests will have same effect.

Reply:
The board should have entropy injected before it gets inserted into the box at the factory. Then, OOB should work.

Reply:
i tend to think of it as our internal tests. so we need to simulate the injection

Reply:
For testing entropy injection itself, yes. For testing everything else, no. Tests should be able to assume a properly manufactured board.

Anyway, nothing to do with this PR. We'll sort this out later.